functions/Get-DbaSqlBuildReference.ps1

```diff
@@ -105,13 +116,26 @@ Function Get-DbaSqlBuildReference {
 				$module_time = Get-Date $module_content.LastUpdated
 				$data_time = Get-Date $data_content.LastUpdated
 				
+				$offline_time = $module_time
 				if ($module_time -gt $data_time) {
 					Copy-Item -Path $orig_idxfile -Destination $writable_idxfile -Force -ErrorAction Stop
 					$result = $module_content
-				}
-				
-				else {
+				} else {
```

Comment:
will adhere but I lost where this is formalized ^_^ pretty recently edited files never had this explicit guideline